Makefile

```diff
@@ -25,9 +25,13 @@ TEST_RESULTS_DIR=testResults
 GOMETALINTER_OPTION=--tests --disable-all -E gofmt -E vet -E golint
 
 IMAGE_REGISTRY ?= local
-K8S_VERSION ?= v1.14.0
+K8S_VERSION ?= v1.14.0-rc.1
```

Comment:
why it's still `v1.14.0-rc.1`?

Reply:
Please rollback this line, v1.14.0 should be used here

Reply:
Okay Sir I am updating that